lib/templates/fragments/header.php

```diff
@@ -47,91 +53,120 @@ function beans_favicon() {
 
 	$url = file_exists( get_stylesheet_directory() . '/favicon.ico' ) ? get_stylesheet_directory_uri() . '/favicon.ico' : BEANS_URL . 'favicon.ico';
 
-	beans_selfclose_markup_e( 'beans_favicon', 'link', array(
-		'rel'  => 'Shortcut Icon',
-		'href' => $url, // Automatically escaped.
-		'type' => 'image/x-icon',
-	) );
-
+	beans_selfclose_markup_e(
+		'beans_favicon',
+		'link',
+		array(
+			'rel'  => 'Shortcut Icon',
+			'href' => $url, // Automatically escaped.
+			'type' => 'image/x-icon',
+		)
+	);
 }
 
 beans_add_smart_action( 'wp_head', 'beans_header_image' );
 /**
  * Print the header image css inline in the header.
  *
  * @since 1.0.0
+ *
+ * @return void
  */
 function beans_header_image() {
 
-	if ( ! current_theme_supports( 'custom-header' ) || ! ( $header_image = get_header_image() ) || empty( $header_image ) ) {
+	$header_image = get_header_image();
```

Comment:
If the theme does not support the header image, there's no need to waste the extra processing time. Why? It's going to bail out. Right?

Therefore, let's put back the order like this:

```
if ( ! current_theme_supports( 'custom-header' ) ) {
     return;
}

$header_image = get_header_image();

if ( empty( $header_image ) ) {
     return;
}
```

Notice too that we can get rid of the `! $header_image` as it's the same as `empty`.